build_tools/mako/configuration.py

```diff
@@ -270,7 +282,7 @@ def get_s20_default_target_list(skipped_target=None,
                 name="S20",
                 benchmark_key="4636549841944576",
                 targets=get_s20_default_target_list(
-                    skipped_target=['cpu', 'vmla', 'cpu2', 'vlk2'])),
+                    skipped_target=['cpu', 'vmvx', 'cpu2', 'vlk2'])),
```

Comment:
This should be `vmvx3t`. (the variable name is confusing, the target means benchmark target, ie, mako_tag, in this case...).

Reply:
ah - gotcha